### ISSUE TITLE
EC2 tags missing on v3.10.2

### ISSUE THREAD

**Issue by jbarnette** [buildkite/agent#968]
Hi yall, I think v3.10.2 has a regression. I just saw a few nodes with `-tags-from-ec2-tags` come up with no tags set. When I rolled them back to v3.10.1 the tags came back.

cc #967

**Comment by jbarnette** [comment]
Nodes are CentOS 7 running the official agent packages. From `buildkite-agent.cfg`:

```
# Tags for the agent (default is "queue=default")
# tags="key1=val2,key2=val2"

# Include the host's EC2 meta-data as tags (instance-id, instance-type, and ami-id)
tags-from-ec2=true

# Include the host's EC2 tags as tags
tags-from-ec2-tags=true

# Include the host's Google Cloud instance meta-data as tags (instance-id, machine-type, preemptible, project-id, region, and zone)
# tags-from-gcp=true

# Include the host's Google Cloud instance labels as tags
# tags-from-gcp-labels=true
```

**Comment by lox** [comment]
Apologies @jbarnette, investigating now and we'll get a fix out asap!

**Comment by lox** [comment]
@jbarnette we found a bug related to `--tags-from-host` in https://github.com/buildkite/agent/issues/967, do you have that enabled too?

**Comment by lox** [comment]
Nevermind, found this one too. Apologies again folks!

**Comment by lox** [comment]
Fixed in v3.10.3

**Comment by jbarnette** [comment]
@lox Thanks for the quick fix! :heart:
